### ISSUE TITLE
Permissions inheritance and precedence

### ISSUE THREAD

**Issue by MisinformedDNA** [MicrosoftDocs/vsts-docs#4052]
For our release pipeline, we would like to limit changes to Release Administrators. We opened the permissions for Contributors on the pipeline and they all were inheriting the "Allow" permission, so we marked them all as "Deny" as saved. Technically, it worked, but it also caused me, a Release Administrator, to lose access as well. Why are is the Contributor role able to override the Release Administrator role?

Here are some images of the permissions:

![image](https://user-images.githubusercontent.com/1784452/56605923-c423f200-65ca-11e9-8168-5687274602aa.png)

![image](https://user-images.githubusercontent.com/1784452/56606143-31d01e00-65cb-11e9-89a3-d962f83cfeaf.png)

![image](https://user-images.githubusercontent.com/1784452/56606074-16fda980-65cb-11e9-9648-05b5764201a7.png)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a8c6f274-a008-fac6-7d99-13d2ce7c8dc3
* Version Independent ID: 916fef9e-d644-2114-b466-e5bda8028ee9
* Content: [Build and release permissions and security roles - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/policies/permissions?view=azure-devops#release-permissions)
* Content Source: [docs/pipelines/policies/permissions.md](https://github.com/MicrosoftDocs/vsts-docs/blob/master/docs/pipelines/policies/permissions.md)
* Product: **devops**
* Technology: **devops-cicd**
* GitHub Login: @andyjlewis
* Microsoft Alias: **alewis**

**Comment by drewlsvern** [comment]
I am experiencing the exact same issue as @MisinformedDNA. Is there a resolution for this?

**Comment by MisinformedDNA** [comment]
@draevin I found [a resolution](https://docs.microsoft.com/en-us/azure/devops/organizations/security/about-permissions?view=azure-devops#permission-settings). For Contributors, turn off inheritance and instead of "Deny", you want "Not set". For Release Administrators, turn off inheritance and make sure everything is set to Allow (maybe this step isn't needed, but it's where I ended up). The key things to know is that 

- One "Deny" takes precedence over any "Allow"
- "Not set" is am implicit deny

![image](https://user-images.githubusercontent.com/1784452/58253663-8ed80480-7d2e-11e9-804f-37abb82d02bc.png)

![image](https://user-images.githubusercontent.com/1784452/58253728-afa05a00-7d2e-11e9-91b8-77736678c59b.png)
